todo_app/templates/index.html

```diff
@@ -9,17 +9,24 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>Todo</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
-        {% for item in items %}
-          <li>{{ item.title }}</li>
+        {% for todo in todos %}
+          <li>{{ todo.title }}
+          <form action={{url_for("complete_card", id=todo.id)}} class"complete-todo-form">
+            <div class="complete-todo-form">
+              <input type="submit" value "mark as completed">
+            </div>
+          </form>
+          </li>
```

Comment:
On the phrase, unfortunately it's just a tiny typo that's really confusing the HTML!
Where you have the value, you're missing an `=`, so the line should be:
```html
<input type="submit" value="mark as completed">
```
which should get the label to show up correctly.

To move it in line, probably the easiest is to wrap the list item inside the form, rather than vice versa:
```html
          <form action={{url_for("complete_card", id=todo.id)}} class"complete-todo-form">
            <li>{{ todo.title }}
              <input type="submit" value="mark as completed">
            </li>
          </form>
```